lib/python/vdm/server/HTTPListener.py

```diff
@@ -480,6 +483,41 @@ def map_deployment_users(request, user):
     return deployment_user[0]
 
 
+def ignore_sighup():
+    signal.signal(signal.SIGHUP, signal.SIG_IGN)
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+
+def start_local_server(deploymentcontents):
+    filename = os.path.join(PATH, 'deployment.xml')
+    deploymentfile = open(filename, 'w')
+    deploymentfile.write(deploymentcontents)
+    deploymentfile.close()
+    voltdb_dir = os.path.realpath(os.path.join(MODULE_PATH, '../../../..', 'bin'))
+    voltdb_cmd = [ 'nohup', os.path.join(voltdb_dir, 'voltdb'), 'create', '-d', filename ]
+    outfilename = os.path.join(PATH, 'voltserver.output')
```

Comment:
Right. Let me copy-paste what I slacked you - 

May be that is somethign we can do later?

I think we should really check if the server is running already and all that before trying to start it again.
So, may be after beta we will fix all that?